src/features/bundlr/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { Flex, TabList, TabPanel, TabPanels, Tabs, Text } from '@chakra-ui/react';
+import { BigNumber } from 'ethers';
+import { useBundlr } from 'features/embalm/stepContent/hooks/useBundlr';
+import { useBundlrSession } from 'features/embalm/stepContent/hooks/useBundlrSession';
+import { SarcoTab } from 'features/sarcophagi/components/SarcoTab';
+import { useSelector } from 'store/index';
+import { BundlrAction, BundlrProfile } from './BundlrProfile';
+import { BusyIndicator } from './BusyIndicator';
+
+export function Bundlr() {
+  const { connectToBundlr } = useBundlrSession();
+  const { fund, withdraw, isFunding, isWithdrawing } = useBundlr();
+  const bundlr = useSelector(s => s.bundlrState.bundlr);
+
+  function handleDeposit(amount: string) {
+    fund(amount);
+  }
+
+  function handleWithdraw(amount: string) {
+    withdraw(BigNumber.from(amount));
+  }
+
+  function handleConnect() {
+    connectToBundlr();
+  }
+
+  return (
+    <Flex
+      w="100%"
+      border="1px solid"
+      borderColor="whiteAlpha.300"
+      direction="column"
+    >
+      <Flex
+        justify="center"
+        w="100%"
+        bg="whiteAlpha.400"
+        py={3}
+      >
+        <Text fontSize="md">BUNDLR</Text>
+      </Flex>
+      <Tabs
+        variant="enclosed"
+        overflow="hidden"
+        isFitted
+        display="flex"
+        flexDirection="column"
+        border="1px solid"
+        borderColor="whiteAlpha.300"
+      >
+        <TabList border="none">
+          <SarcoTab>Deposit</SarcoTab>
+          <SarcoTab>Withdraw</SarcoTab>
+        </TabList>
+        <TabPanels
+          overflow="hidden"
+          bg="linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0.09) 100%);"
+        >
+          <TabPanel
+            px={10}
+            py={6}
+            h="100%"
+          >
+            {isFunding ? (
```

Comment:
A user will be unable to interact with the deposit tab while funding and will be unable to interact with the withdraw tab while withdrawing. However, a user may withdraw while funding or fund while withdrawing.

Reply:
One note related to this --- there is a `pendingBalance / setPendingBalance` state on the global `bundlrState` --- 

This is used to detect if a user has completed funding TX but it hasn't yet showed up in their balance. We could use that here to show whether the user has a pending balance in the UX (not sure if we have any designs for that).

This pending state will get set on funding, but doesn't yet handle withdrawals or multiple TXs. 

In a separate PR, we will want to get that updated to handle the withdrawal delay as well as potentially handle an array of TXs.